PathPlanning/DynamicWindowApproach/dynamic_window_approach.py

```diff
@@ -152,13 +149,18 @@ def calc_control_and_trajectory(x, dw, config, goal, ob):
                 min_cost = final_cost
                 best_u = [v, y]
                 best_trajectory = trajectory
-
+                if abs(best_u[0]) < 0.001 and abs(x[3]) < 0.001:
```

Comment:
Could you please create two class const variable for 0.001 and 0.001 with proper variable name?

Reply:
No problem, the 0.001 is changed to config.robot_stuck_flag_cons.